apl-conf/src/main/resources/conf-tn3/apl-blockchain.properties

```diff
@@ -118,7 +118,7 @@ apl.pullThreshold=0
 apl.blacklistingPeriod=600000
 
 # Consider a new transaction or block sent after 10 peers have received it.
-apl.sendToPeersLimit=10
+apl.sendToPeersLimit=3
```

Comment:
As I see property has been changed only for tn-3. Are you going to change it for the tn-1/2 and mainnet?

Reply:
Fixed